config/locales/en.yml

```diff
@@ -106,6 +106,7 @@ en:
       enabled: Enabled
       feed_url: Feed URL
       followers: Followers
+      followers_local: "(%{local} local)"
```

Comment:
This is not used

Reply:
It is added by executing "bundle exec i18n-tasks add-missing".
Is it safe to delete it?